tests/monotouch-test/Foundation/NSKeyedUnarchiverTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Foundation;
+using NUnit.Framework;
+using ObjCRuntime;
+
+namespace MonoTouchFixtures.Foundation
+{
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class NSKeyedUnarchiverTest
+	{
+		[Test]
+		public void GetUnarchivedObject_TypeWrappers ()
+		{
+			if (!TestRuntime.CheckXcodeVersion (10, 0))
+				Assert.Ignore ("Ignoring GetUnarchivedObject_TypeWrappers as requires new APIs");
+
+			NSDictionary<NSString, NSString> testValues = new NSDictionary<NSString, NSString> ((NSString)"1", (NSString)"a");
+			NSData data = NSKeyedArchiver.ArchivedDataWithRootObject (testValues, true, out NSError error);
+			Assert.IsNull (error);
```

Comment:
name the asserts, too many reports (and sometimes bad symbols) makes debugging failures harder than it have to be